core/constants.yaml

```diff
@@ -165,7 +165,7 @@ qanet_tns: &qanet_tns
 travis: &travis
   <<: *qanet_tns
 
-  k: 3  # it's needed for tests
+  k: 300  # it's needed for tests
```

Comment:
How else would the introduced test work? The amount of generated blocks cannot exceed `k`, and `3` is too little. I'll need at least 10 or 20 here for the test to make sense.